NOTICE

```diff
@@ -191,13 +191,6 @@ cache-api:
   * HOMEPAGE:
     * https://github.com/jsr107/jsr107spec
 
-easymock

```

Comment:
NOTICE and LICENSE has been changed according to ASF policy. Generally you don't have change these two files.